### ISSUE TITLE
About loss

### ISSUE THREAD

**Issue by joyyang1997** [albertpumarola/GANimation#84]
Hi,
it seems that you ignore the fake image condition  loss: E[||D(G(I|y)) - y||^2], is this better for training?
```
# D(fake_I)
d_fake_desired_img_prob, **_** = self._D.forward(fake_imgs_masked.detach())
self._loss_d_fake = self._compute_loss_D(d_fake_desired_img_prob, False) * self._opt.lambda_D_prob
```

**Comment by vipermu** [comment]
In this step you're training the Discriminator in the case when you have a fake image in the input. You don't want to update the Action Units classification with fake images.
